lms/templates/navigation/bootstrap/navbar-authenticated.html

```diff
@@ -56,8 +56,10 @@
 
   <ul class="navbar-nav navbar-right">
     % if should_display_shopping_cart_func() and not (course and static.is_request_in_themed_site()): # see shoppingcart.context_processor.user_has_cart_context_processor
-      <a role="button" class="nav-item-open-collapsed btn-shopping-cart btn btn-secondary mr-3" href="${reverse('shoppingcart.views.show_cart')}">
-        <span class="icon fa fa-shopping-cart" aria-hidden="true"></span> ${_("Shopping Cart")}
+      <a href="${reverse('shoppingcart.views.show_cart')}">
```

Comment:
Sounds good. For future reference, this is the case even if the button is wrapped by a link and we don't have a button onclick event?